website/docs/components/table/partials/specifications/anatomy.md

```diff
@@ -2,20 +2,22 @@
 
 ### Table headers
 
-![Table header anatomy](/assets/components/table/table-header-anatomy.png)
+![Table header selection anatomy](/assets/components/table/table-header-select-anatomy.png)
 
 | Element          | Usage                                           |
 |------------------|-------------------------------------------------|
+| Checkbox         | Optional, but required when cells yield a checkbox |
```

Comment:
The image says "Selection", but the table says "Checkbox". How has the team been referring to this element in discussions? I'd probably lean toward "Checkbox" personally, since that's the name of the nested component.

It's important to use consistent naming in the anatomy section (really everywhere) as it helps everyone use the same naming conventions/terminology when talking about the parts of a component.